### ISSUE TITLE
Request: Support non-abbreviated prop names.

### ISSUE THREAD

**Issue by tasteee** [kuma-ui/kuma-ui#419]
I am really into Kuma right now. It seems to go against most of my deeply held beliefs about separation of concerns and variability, but I think with a decade of experience doing this shit now I am starting to find value in tools that allow you the flexibility to produce solutions with your prophetic, purist ideas while offering the flexibility to do pretty much as you please.

But one thing I am on about that I have a hard time compromising on is naming things.

My entire career I have focused on developer experience, so I am hyper sensitive to cases in which code opens the door for uncertainty or for questions to be asked. For at least 6 years I have always opted for `index` over `i`, `event` over `e`, so on and so forth.

The point is to accomodate anybody of 1) any experience level 2) any part of the world 3) any cultural background -- only knowing for certain that (at least for a dev like myself working on teams in the US) the devs that read / maintain this code will have baseline familiarity with English -- nothing more.

My efforts are all about reducing cognitive load and setting up future developers for utmost success.

```js
// interrogatives -- ask a question with a yes or no answer.
isGood
didFooBar
canUserSave
willWeEverKnow
shouldBreakLegs
hasBeenDone

// "sssssss' -- pluralsssss. obviously plural means 2, and if you have more than 1
// of anything in JS, well, they must be in some sort of iterable.
cats
users
olympians
configurations
tyrannosaurusRexs

// modifier word to indicate it is a number.
catCount
trexWeight
snakeLength
songDuration

// objects are things. so just name the object what it represents. bada-bing, bada-boom.
// youd neve expect cat to be a string, a boolean, an array, a map, a number... a cat is a thing.
user 
cat
authentication
configuration

// so on and so forth. The code itself should be intuitively written, but naming conventions are just an added helper.
```

So when I look at props named `p pt pr pb pl m mt mr mb ml` etc, I can't help but to think that this must be... just to save keystrokes? 

But, everything we do in code is a pattern.

And everything we do in code is a vote for how to handle things in the future. For me to embrace these patterns in my code is an involuntary vote for shorthand to be used elsewhere. It opens the door for there to be an argument to abbreviate other things. And I try not to play that game -- and I abstain with good intent.

Longwinded, my apologies. But my ask is whether or not you'd be open to supporting the longhand variations of these shorthand prop names, as well?

Like I said, I am starting to find value in tools that allow you the flexibility to produce solutions with your prophetic, purist ideas while offering the flexibility to do pretty much as you please. And this is one incompatibility between my phophetic, purist ideas and Kuma. 😁

**Comment by poteboy** [comment]
Yes, I'd be more than happy to support this. If you could send a PR with the longhand variations, I'll make sure to review and support it 🙏 The related code is around @kuma-ui/system.

**Comment by tasteee** [comment]
Yup, I would love to provide a PR for that! :)

Thank you so much for being open minded!
